chapters/image_accceses.md

```diff
@@ -0,0 +1,146 @@
+# Image Accesses
+
+SPIR-V has various [Image Instructions](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_image_instructions) that interact with `OpTypeImage`. This chapter aims to give some extra context around these instructions.
+
+> There is an [example.frag](./examples/image_accesses/example.frag) (and [example.spv](./examples/image_accesses/example.spv)) GLSL shader that has an example of various ways images can be accessed in SPIR-V.
+
+# Coordinate Systems
+
+It is important to know there are 3 texel coordinate systems used for images:
```

Comment:
I am not sure where it fits (and perhaps it doesn't fit in this document), but just as the coordinates can use ints and floats (which may be normalized or not), I was curious about the texel data (which I know can be ints or floats). Is there any way to know whether float texel data is normalized?

Reply:
I assume the Format itself could define that, like if you look at `VK_FORMAT_A2B10G10R10_UNORM_PACK32` it is defined as

>  32-bit packed unsigned normalized format

and from there is mapped to the SPIR-V format 

https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#spirvenv-image-formats

![image](https://github.com/user-attachments/assets/1738bcd0-09c8-4f4d-bb3f-c6574b9269fb)